docs/topics/dokka-migration.md

```diff
@@ -148,21 +147,25 @@ options according to your project setup:
   documentedVisibilities.set(
       setOf(VisibilityModifier.Public)
   )
+  // or
+  documentedVisibilities(VisibilityModifier.Public)
   ```
 
 * **External documentation link:** The source link configuration has [changed](https://docs.gradle.org/current/userguide/upgrading_version_8.html#deprecated_invalid_url_decoding). 
     The remote URL is now specified using the `URI` class instead of the `URL` class.
 
   Previous configuration:
   
-    ```kotlin
-    remoteUrl.set(URL("https://github.com/your-repo"))   
+    ```text
+    remoteUrl.set(URL("https://github.com/your-repo"))
     ```
```

Comment:
@AlejandraPedroza I see you changed this code code block to `text`, but I don't understand why, so it looks like a mistake. Could you check please?

Reply:
Fixed.